command/command.go

```diff
@@ -297,9 +297,6 @@ func handleCommitRequest(cache *cache.Cache, commitMsg *sync_pb.CommitMessage, c
 		entryRsp.ResponseType = &rspType
 		entryRsp.IdString = aws.String(entityToCommit.ID)
 		entryRsp.Version = entityToCommit.Version
-		entryRsp.ParentIdString = entityToCommit.ParentID
```

Comment:
why we had to remove it?

Reply:
I updated protobuf schemas from chromium.  This field was updated to be marked reserved and was dropped from the generated Go code.